packages/angular/src/generators/application/files/ng-module/src/app/app.component.spec.ts__tpl__

```diff
@@ -0,0 +1,33 @@
+import { TestBed } from '@angular/core/testing';
+import { AppComponent } from './app.component';<% if(!minimal) { %>
+import { NxWelcomeComponent } from './nx-welcome.component';<% } %><% if(routing) { %>
+import { RouterTestingModule } from '@angular/router/testing';<% } %>
+
+describe('AppComponent', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [<% if(routing) { %>RouterTestingModule<% } %>],
+      declarations: [AppComponent<% if(!minimal) { %>, NxWelcomeComponent<% } %>]
+    }).compileComponents();
+  });
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
+  it(`should have as title '<%= appName %>'`, () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app.title).toEqual('<%= appName %>');
+  });
+  <% if(!minimal) { %>
+  it('should render title', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    fixture.detectChanges();
+    const compiled = fixture.nativeElement as HTMLElement;
+    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
+  });
+  <% } %>
```

Comment:
Let's remove the useless `should create the app` test case and rearrange the rest based on proposed changes to the component generation:
```suggestion
  it('should render title', () => {
    const fixture = TestBed.createComponent(AppComponent);
    fixture.detectChanges();
    const compiled = fixture.nativeElement as HTMLElement;
    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
  });<% if(!minimal) { %>

  it(`should have as title '<%= appName %>'`, () => {
    const fixture = TestBed.createComponent(AppComponent);
    const app = fixture.componentInstance;
    expect(app.title).toEqual('<%= appName %>');
  });<% } %>
```